platform/remarkable/koreader.sh

```diff
@@ -1,12 +1,21 @@
 #!/bin/sh
 export LC_ALL="en_US.UTF-8"
-
 # working directory of koreader
 KOREADER_DIR="${0%/*}"
 
 # we're always starting from our working directory
 cd "${KOREADER_DIR}" || exit
 
+# reMarkable 2 check
+IFS= read -r MACHINE_TYPE < "/sys/devices/soc0/machine"
+if [ "reMarkable 2.0" = "$MACHINE_TYPE" ]; then
+    if [ -z $RM2FB_SHIM ]; then
```

Comment:
ShellCheck would complain about that, and well, I've experienced enough later mysterious minor issues due to thinking quoting shouldn't be necessary in a particular spot that I think it's definitely a good habit to always do it (unless you do want to split on spaces, of course).
```suggestion
    if [ -z "$RM2FB_SHIM" ]; then
```